packages/cardano-services/src/ChainHistory/DbSyncChainHistoryProvider/types.ts

```diff
@@ -0,0 +1,153 @@
+import { Cardano } from '@cardano-sdk/core';
+
+export type TransactionDataMap<T> = Map<Cardano.TransactionId, T>;
+export type TxOutTokenMap = Map<string, Cardano.TokenMap>;
+export type TxTokenMap = TransactionDataMap<Cardano.TokenMap>;
+
+export interface BlockModel {
+  block_no: number;
+  epoch_no: number;
+  epoch_slot_no: number;
+  hash: Buffer;
+  next_block: Buffer | null;
+  previous_block: Buffer | null;
+  size: number;
+  slot_leader_hash: Buffer;
+  slot_leader_pool: string | null;
+  slot_no: number;
+  time: string;
+  tx_count: string;
+  vrf: string;
+}
+
+export interface BlockOutputModel {
+  fees: string;
+  output: string;
+  hash: Buffer;
+}
+
+export interface TipModel {
+  block_no: number;
+  hash: Buffer;
+  slot_no: number;
+}
+
+export interface TxModel {
+  id: Buffer;
+  index: number;
+  size: number;
+  fee: string;
+  invalid_before: string | null;
+  invalid_hereafter: string | null;
+  block_no: number;
+  block_hash: Buffer;
+  block_slot_no: number;
+}
+
+export interface TxInOutModel {
+  address: string;
+  coin_value: string;
+  datum?: Buffer | null;
+  id: string;
+  index: number;
+  tx_id: Buffer;
+}
+
+export interface TxOutput extends Cardano.TxOut {
```

Comment:
```suggestion
export interface TxOutputModel extends Cardano.TxOut {
```
?

Reply:
Done in 70dd1e23cbb08257e051305e215dc03c001d0d99